common/tests/trusted/enclave/CMakeLists.txt

```diff
@@ -57,6 +63,6 @@ TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} -Wl,-L,${SGX_SSL}/lib64/release)
 TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} -Wl,-L,${PDO_TOP_DIR}/common/build)
 TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} -Wl,--whole-archive -lsgx_tsgxssl -Wl,--no-whole-archive)
 TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} -Wl,--whole-archive -l${TRTS_LIBRARY_NAME} -Wl,--no-whole-archive)
-TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} -Wl,--start-group -l${TRUSTED_LIB_NAME} -lsgx_tsgxssl_crypto -lsgx_tstdc -lsgx_tcxx -lsgx_tcrypto -l${SERVICE_LIBRARY_NAME} -Wl,--end-group)
+TARGET_LINK_LIBRARIES(${ENCLAVE_NAME} "-Wl,--start-group" "${TRUSTED_LIB_NAME}" "${T_CRYPTO_LIB_NAME}" "sgx_tsgxssl_crypto" "sgx_tstdc" "sgx_tcxx" "sgx_tcrypto" "${SERVICE_LIBRARY_NAME}" "-Wl,--end-group")
```

Comment:
why are you adding the quotes?

Reply:
mainly to remove the `-l`. Not a big deal, can revert.

Reply:
same exact number of characters :)

Reply:
quotes removed